playbooks/openstack-create_network.yml

```diff
@@ -19,27 +19,70 @@
             parameters:
               prefix: "{{ socok8s_envname }}"
               network_external: "{{ deploy_on_openstack_external_network }}"
+              network_internal_cidr: "{{ deploy_on_openstack_internal_subnet_cidr }}"
 
         - name: show network stack output
           debug:
             var: network_stack_create_output
 
-    - name: Get the internal VIP from the network stack
-      command: openstack stack output show {{ deploy_on_openstack_network_stackname }} dcm_vip -c output_value -f value
-      changed_when: false
-      register: _network_stack_dcm_vip_output
+    - name: Add vip with cidr in extravars
+      delegate_to: localhost
+      blockinfile:
```

Comment:
I like the blockinfile because it adds this nice marker so you know that theses lines are edited by ansible (you don't know that with `lineinfile`

Reply:
ok for me.

Reply:
Could we combine all of the openstack IDs into a single block? That would give us better signal-to-noise in the target file since each block comes with two comment lines.

Reply:
That doesn't work because we loop over all output entries for every block.

Reply:
Well we could technically have a marker, and then add lineinfiles based on previous expected lines. But it's not important for me.